contribs/gnodev/pkg/browser/model_style.go

```diff
@@ -0,0 +1,239 @@
+package browser
+
+import "github.com/charmbracelet/glamour/ansi"
+
+const (
+	defaultListIndent      = 2
+	defaultListLevelIndent = 4
+	defaultMargin          = 2
+)
+
+// Catpuccin style: https://github.com/catppuccin/catppuccin
+// XXX: update this with `gno` colors scheme
```

Comment:
I agree in the future the various `color` hardcoded can be replaced with some styles coded in constants